crates/subspace-runtime/src/lib.rs

```diff
@@ -1047,6 +1049,10 @@ impl_runtime_apis! {
         fn extract_root_blocks(ext: &<Block as BlockT>::Extrinsic) -> Option<Vec<RootBlock>> {
             extract_root_blocks(ext)
         }
+
+        fn extract_block_object_mapping(block: Block) -> BlockObjectMapping {
+            extract_block_object_mapping(block, vec![])
```

Comment:
This would have been even simpler as we know no mappings will be extracted in practice:
```suggestion
            BlockObjectMapping::default()
```